src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/csip/CSIPMappingResourceIntTest.kt

```diff
@@ -638,6 +644,133 @@ class CSIPMappingResourceIntTest : IntegrationTestBase() {
     }
   }
 
+  @DisplayName("DELETE /mapping/csip/dps-csip-id/{dpsCSIPId} with children")
```

Comment:
not that matters but I used `all` in urls where I would delete children in there mapping endpoints

Reply:
that's a good idea - will update in another PR